wallet-mobile-app-centbit/Constants/colors.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+
```

Comment:
this is really goood, you should have for size, font and event strings incase on int18